Bottom: 25795c71de349a280df948e9098a56a50c76435e
Top:    c096dcff0d5d97207fb65f8ca607b8a29148e67e
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-07-18 14:05:01 -0700

Refresh of dbg

---

diff --git a/src/rsocket.c b/src/rsocket.c
index 003b7d8..68361f5 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -425,10 +425,12 @@ static int rs_create_ep(struct rsocket *rs)
 
 	rs_set_qp_size(rs);
 	ret = rs_init_bufs(rs);
+	printf("rs create ep init bufs %d\n", ret);
 	if (ret)
 		return ret;
 
 	ret = rs_create_cq(rs);
+	printf("rs create ep - create cq %d\n", ret);
 	if (ret)
 		return ret;
 
@@ -445,6 +447,7 @@ static int rs_create_ep(struct rsocket *rs)
 	qp_attr.cap.max_inline_data = rs->sq_inline;
 
 	ret = rdma_create_qp(rs->cm_id, NULL, &qp_attr);
+	printf("rs create ep - create qp %d\n", ret);
 	if (ret)
 		return ret;
